packages/core/contracts/oracle/implementation/VotingV2.sol

```diff
@@ -0,0 +1,1102 @@
+// TODO: this whole /oracle/implementation directory should be restructured to seperate the DVM and the OO.
```

Comment:
this contract is mostly a copy-paste of the main `Voting.sol` contract except for the addition of some untested slashing functions and some todos on things that will be changed in the DVM-v2 implementation. for now it can be mostly ignored in review